packages/ui/src/components/InvoicesOverview/InvoicesOverview.tsx

```diff
@@ -47,43 +47,49 @@ export const InvoicesOverview: React.FC<{
     );
 
   return (
-    <div className="w-full grid gap-6 flex-wrap grid-cols-[repeat(auto-fill,minmax(310px,auto))] [&>*]:max-w-none">
+    <div className="w-full grid grid-cols-4 [&>*]:col-span-2 md:[&>*]:col-span-1 gap-4 lg:gap-6 flex-wrap">
```

Comment:
Please put the classname on the element itself. There is no need to do it this way.